apps/src/componentLibrary/tabs/tabs.module.scss

```diff
@@ -35,85 +48,201 @@
 .tabs-primary {
   li button {
     align-items: baseline;
-
-    color: $light_gray_700;
-
     background-color: transparent;
     border-radius: 0;
+  }
 
-    &:hover {
-      color: $light_primary_500;
-    }
+  &.tabs-light {
 
-    &:focus-visible {
-      color: $light_primary_500;
-      outline: 2px solid $light_primary_500;
+    ul::after {
+      border-color: $light_gray_200;
     }
 
-    &:active:not(:disabled) {
-      color: $light_primary_500;
-      border: none !important;
-      border-bottom: 2px solid $light_primary_500 !important;
+    li button {
+      color: $light_gray_800;
+
+
+      &:hover {
+        color: $light_primary_500;
+        border: none;
+        border-bottom: 2px solid $light_primary_500;
```

Comment:
On hover, I see that the bottom border doesn't quite extend all the way to the left and right.
![Screenshot 2024-07-10 at 1 25 11 PM](https://github.com/code-dot-org/code-dot-org/assets/107423305/fcc1ba2c-90a7-441e-8b55-896483cff593)

As a comparison, here's what it looks like without bottom border on hover. I think maybe this is a question for @markabarrett ?

![Screenshot 2024-07-10 at 1 30 20 PM](https://github.com/code-dot-org/code-dot-org/assets/107423305/667466a5-e7c1-4c04-b7fc-e83044ba1842)